### PR TITLE
Changed Kubernetes version to 1.20.15 and updated patches

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Removed unneeded patches
* Removed some lines in patches that we're apparently no longer going to update

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
Checksum will fail. I'll update them

File: projects/kubernetes/kubernetes/1-20/patches/0002-EKS-PATCH-Pass-region-to-sts-client-for-ap-east-1.patch [hunk 1]
```diff
@@ -119,6 +119,3 @@ index fbbaa6567d0..6f280c19320 100644
  	regionName, err := azToRegion(zone)
  	if err != nil {
  		return nil, err
```
Comment (TerryHowe) [inline projects/kubernetes/kubernetes/1-20/patches/0002-EKS-PATCH-Pass-region-to-sts-client-for-ap-east-1.patch:121]: Are making these with `git format-patch` or did you manually just remove this trailer (which I thought was the git version)? Same with the  "From" I thought was the previous commit, does that make it a wild card?

Comment (kschumy) [reply projects/kubernetes/kubernetes/1-20/patches/0002-EKS-PATCH-Pass-region-to-sts-client-for-ap-east-1.patch:121]: We're doing something different for this version. I'll slack you

**Review by jaxesn — CHANGES_REQUESTED** [review]

**Review by jaxesn — APPROVED** [review]
/lgtm

**Pull request merged**